src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -283,7 +283,7 @@ In order to debug this project, add an executable project to this solution which
     <value>Project</value>
   </data>
   <data name="PartialSupportedDotNetCoreProject" xml:space="preserve">
-    <value>Visual Studio 2017 version 15.7 or newer is recommended for .NET Core {0}.{1} projects.</value>
+    <value>A newer version of Visual Studio is recommended for projects targetting .NET Core {0}.{1}.</value>
```

Comment:
"targeting"

Reply:
Oh you copied https://webpifeed.blob.core.windows.net/webpifeed/Partners/DotNetVersionCompatibility.json, @BillHiebert do you own that?